core/include/moveit/task_constructor/task.h

```diff
@@ -105,7 +105,7 @@ class Task : protected WrapperBase
 	/// add function to be called after each top-level iteration
 	TaskCallbackList::const_iterator addTaskCallback(TaskCallback&& cb);
 	/// remove function callback
-	void erase(TaskCallbackList::const_iterator which);
+	void eraseTaskCallback(TaskCallbackList::const_iterator which);
```

Comment:
IMHO, different variants of `erase` can be distinguished by their argument. But I don't mind.